learning_observer/learning_observer/google.py

```diff
@@ -40,8 +36,11 @@
 import learning_observer.runtime
 import learning_observer.prestartup
 
+from learning_observer.lms_integration import BaseLMS, Endpoint, register_cleaner
```

Comment:
The cleaner way to present this is
```python
import learning_observer.lms_integration
...
GOOGLE_ENDPOINTS = list(map(lambda x: learning_observer.lms_integration.Endpoint(...
```
avoids any duplicate variable name bugs in the future and immediately informs the user where the function they are using comes from.

Reply:
Done